examples/README.md

```diff
@@ -25,13 +25,13 @@ This directory contains examples of plots that use Bokeh's [compatibility layer]
 
 This directory includes examples that show how to embed Bokeh plots in as DOM objects within an HTML document.
```

Comment:
> This directory includes examples that show how to embed Bokeh plots in as DOM objects within an HTML document.

propose change to

This directory includes examples that show how to embed Bokeh plots in an HTML document.